sys/usb/usbus/usbus_control.c

```diff
@@ -307,6 +307,17 @@ static void _usbus_config_ep0(usbus_control_handler_t *ep0_handler)
     usbdev_ep_ready(ep0_handler->out, 0);
 }
 
+uint8_t *usbus_control_get_out_data(usbus_t *usbus, size_t *len)
+{
+    assert((int)usbus->state == (int)USBUS_CONTROL_REQUEST_STATE_OUTDATA);
```

Comment:
Is it desireable to mix up `usbus_state_t` with `usbus_setuprq_state_t` ?

Reply:
No, definitely not

Reply:
Fixed